addons/web_editor/static/src/js/editor/odoo-editor/src/commands/commands.js

```diff
@@ -317,6 +318,15 @@ export const editorCommands = {
         }
 
         currentNode = lastChildNode || currentNode;
+        if (
+            currentNode.nodeName !== 'BR' &&
+            currentNode.nextSibling &&
+            currentNode.nextSibling.nodeName === 'BR' &&
+            lastLeaf(currentNode.parentNode) === currentNode.nextSibling &&
+            !closestElement(currentNode, '[t-field],[t-esc],[t-out]')
```

Comment:
@dmo-odoo In 17.0 I had to change this to not remove `<br>` elements followed by a qweb node reason without the `<br>` element when two fields are created in studio beside one another.
eg: `<span t-field="...." contenteditable="false">test</span>[]` the anchorNode would be the field itself, so this [code](https://github.com/odoo/odoo/blob/17.0/addons/web_editor/static/src/js/backend/QWebPlugin.js#L136C1-L147C6) would select the qweb node and replace it with the new node.

Reply:
Ok, thanks for the heads up! Do we have a test that cover that particular case? I think we should have one.

Reply:
We already do have a tour in [`web_studio`](https://github.com/odoo/enterprise/blob/17.0/web_studio/tests/test_report_editor.py#L843) that would fail.